llama-index-core/llama_index/core/response_synthesizers/base.py

```diff
@@ -207,19 +235,24 @@ def synthesize(
 
             event.on_end(payload={EventPayload.RESPONSE: response})
 
+        dispatcher.event(SynthesizeEndEvent())
```

Comment:
should the synthesize event be logging the response and query? Previously there as an event logging the query, retrieved nodes, response... just wanted to make sure we aren't dropping information

Reply:
Or maybe the legacy thing is catching that haha not sure

Reply:
Yeah the since removed `LegacyCallbackSpanHandler` was dealing with the payload stuff. 😅

BUT, I do think that we should be adding more payloads to these events. I think its reasonable to add response and query to this `SynthesizeEndEvent` now, at least as an example (even if no one is doing anything with it yet). 

I'll add it.